flac.md

```diff
@@ -191,7 +191,7 @@ Data     | Description
 
 FLAC specifies a minimum block size of 16 and a maximum block size of 65535, meaning the bit patterns corresponding to the numbers 0-15 in the minimum blocksize and maximum blocksize fields are invalid.
 
-The MD5 signature is made by performing an MD5 transformation on the samples of all channels interleaved, represented in a signed, little-endian form. This interleaving is on a per-sample basis, so for a stereo file this means first the first sample of the first channel, then the first sample of the second channel, then the second sample of the first channel etc.
+The MD5 signature is made by performing an MD5 transformation on the samples of all channels interleaved, represented in a signed, little-endian form. This interleaving is on a per-sample basis, so for a stereo file this means first the first sample of the first channel, then the first sample of the second channel, then the second sample of the first channel etc. In case the bit depth is not a whole number of bytes, samples are left padded (right-justified) with zeros to a whole number of bytes before transforming.
```

Comment:
Maybe this is clearer (assuming that it actually matches the reference implementation):

> In case the bit depth is not a whole number of bytes, additional zero bits are inserted at the most-significant position until the bit depth is a multiple of 8. This padding is performed for each channel independently, and before byte encoding.

Reply:
Yes, it does read cleaner. However, what do you mean by byte encoding?

Reply:
I tried to express that the padding happens on the sample values, not the output bytes.